### PR TITLE
Suggestions for few changes in coco_gen dataset

### PR DESCRIPTION
**Pull request by mohitmundhragithub**
These are few suggestions for the app for coco-gen dataset

1. for performance runs, no need to run "processOutput" completely. This could be determined by not passing the model file.
2. input to the clip_model should be in the range -1.0 to +1.0

### CODE REVIEW

File: flutter/assets/tasks.pbtxt [hunk 1]
```diff
@@ -303,18 +303,18 @@ task {
     type: COCOGEN
     full {
       name: "COCO validation set for Stable Diffusion"
-      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_full.tfrecord"
+      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_test.tfrecord"
```
Comment (anhappdev) [inline flutter/assets/tasks.pbtxt:306]: Should it be?
* full -> coco_gen_full (5000 prompts)
* lite -> coco_gen_test (150 prompts)

Comment (mohitmundhragithub) [reply flutter/assets/tasks.pbtxt:306]: I think that's what was decided right? We will use some random samples from full set for performance test.
But use the 150 prompts for measuring accuracy.
But we can double check in the meeting today.

**Comment by mohitmundhragithub** [comment]
@anhappdev @freedomtan i have fixed the path. Please merge this change.

File: flutter/assets/tasks.pbtxt (hunk 1 above)
Comment (anhappdev) [reply flutter/assets/tasks.pbtxt:306]: Yes. You are correct. I forget that we use the lite one for performance test.

**Review by anhappdev — APPROVED** [review]

**Pull request merged**